pkg/cloud/services/elb/loadbalancer_test.go

```diff
@@ -2330,21 +2334,21 @@ func TestGetHealthCheckProtocol(t *testing.T) {
 		{
 			"protocol http",
 			&infrav1.AWSLoadBalancerSpec{
-				HealthCheckProtocol: &infrav1.ELBProtocolHTTP,
```

Comment:
can we please leave some test case using the built-in variables? so we cover both

Reply:
I don't think it makes sense to use the built-in variables. When a user will provide these values via a AWSCluster object, these will always be new variables

Reply:
that's not necessarily true. As a consumer of the API I can programatically choose to vendor capa and use the built-in variable to define my instance of the awsCluster. Just a nit anyway, lgtm.